### PR TITLE
Improve array related styling of renderer

### PR DESCRIPTION
**Pull request by edgarmueller**
Also fixes some minor test-related issues

### CODE REVIEW

**Comment by eneufeld** [comment]
The array renderer looks good. 
For me the nester array example is quite laggy when I type something into the name field, could you recheck whether it is my machine or whether we have a bug somewhere?

File: packages/core/test/util/runtime.test.ts [hunk 1]
```diff
@@ -238,7 +238,7 @@ test('evalEnablement disable invalid case', t => {
     t.is(evalEnablement(uischema, data), true);
 });
 
-test('evalEnablement disable invalid case based on schema condition', t => {
+test.skip('evalEnablement disable invalid case based on schema condition', t => {
```
[the remark this answers is not in the document]
Comment (edgarmueller) [reply packages/core/test/util/runtime.test.ts]: Fixed the previous one, but this on is failing.

**Comment by edgarmueller** [comment]
I don't think this is directly related to the array renderers but rather to the change event being fired too often and for me the 'lagginess' also happens in other examples. We should probably debounce change events, but that should be part of a different issue then.

**Comment by eneufeld** [comment]
Can you open a new issue for the lagginess?

File: packages/core/test/util/runtime.test.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (eneufeld) [reply packages/core/test/util/runtime.test.ts]: did it work before the fix? Is this fixed with the changes in #1148?

[the remark this answers is not in the document]
Comment (edgarmueller) [reply packages/core/test/util/runtime.test.ts]: Should be fixed, the test had the wrong expectations.

**Pull request merged**